bindings/experimental/distrdf/python/DistRDF/_graph_cache.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
```

Comment:
I guess we don't need this, if we are using dataclasses we assume at least Python 3.7

Reply:
Delayed evaluation of annotations is a Python 3.7 feature indeed, but needs to be enforced via the `__future__` import. Unfortunately, the Python standard hasn't yet decided on whether this will actually be the default behaviour (see https://docs.python.org/3/library/__future__.html#id1). In DistRDF we use it extensively both to avoid circular imports and to avoid importing modules that are used only for annotations, so I just write the `__future__` import in every module